readme.md

```diff
@@ -303,19 +303,14 @@ If you run *firmata* from the command line it will prompt you for the serial por
 
 - `board.serialConfig(options)`
 
-  Configure a hardware or serial port
+  Configure a hardware or serial port -- required before using serial read/write functions
 
   ```
-  options {
-    portId {number}
-      The serial port to use
-      (HW_SERIAL1, HW_SERIAL2, HW_SERIAL3, SW_SERIAL0, SW_SERIAL1, SW_SERIAL2, SW_SERIAL3)
-    baud {number}
-      The baud rate of the serial port
-    rxPin {number}
-      [SW Serial only] The RX pin of the SoftwareSerial instance
-    txPin {number}
-      [SW Serial only] The TX pin of the SoftwareSerial instance
+  options = {
+    portId: 0,      // <number> The serial port to use (HW_SERIAL1, HW_SERIAL2... )
```

Comment:
so it can be any value 1-4 or 8-12. The ability to set 0 will be enabled at some point, but only when using a non-Serial transport such as Ethernet, WiFi, BLE, etc.